<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UrlConfigurationDialog</class>
 <widget class="QWidget" name="UrlConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>698</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QFormLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="remoteProtocol">
     <property name="text">
      <string>Access protocol:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="caldav">
     <property name="text">
      <string>CalDA&amp;V</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="carddav">
     <property name="text">
      <string>CardDAV</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="groupdav">
     <property name="text">
      <string>GroupDAV</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Remote URL:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="remoteUrl"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label1">
     <property name="text">
      <string>Access:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QRadioButton" name="useDefaultCreds">
       <property name="text">
        <string>Use global creden&amp;tials</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="useSpecificCreds">
       <property name="text">
        <string>Use specific credentials</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="credentialsGroup">
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="toolTip">
           <string comment="The user's login on the remote server"/>
          </property>
          <property name="text">
           <string>Username:</string>
          </property>
          <property name="buddy">
           <cstring>username</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="username"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="toolTip">
           <string comment="The user's login on the remote server"/>
          </property>
          <property name="text">
           <string>Password:</string>
          </property>
          <property name="buddy">
           <cstring>password</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="KPasswordLineEdit" name="password">
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="collectionsLabel">
     <property name="text">
      <string>Discovered collections:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QTreeView" name="discoveredUrls"/>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="fetchButton">
       <property name="text">
        <string>Fetch</string>
       </property>
      </widget>
     </item>
    </layout>  
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPasswordLineEdit</class>
   <extends>QWidget</extends>
   <header>kpasswordlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>useSpecificCreds</sender>
   <signal>toggled(bool)</signal>
   <receiver>credentialsGroup</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
